controllers/accesslogpolicy_controller.go

```diff
@@ -213,17 +239,12 @@ func (r *accessLogPolicyReconciler) reconcileUpsert(ctx context.Context, alp *an
 		return err
 	}
 
-	err = r.updateAccessLogPolicyStatus(ctx, alp, gwv1alpha2.PolicyReasonAccepted, config.LatticeGatewayControllerName)
-	if err != nil {
-		return err
-	}
-
 	r.eventRecorder.Event(alp, corev1.EventTypeNormal, k8s.ReconciledEvent, "Successfully reconciled")
 
 	return nil
 }
 
-func (r *accessLogPolicyReconciler) targetRefExists(ctx context.Context, alp *anv1alpha1.AccessLogPolicy) (bool, error) {
+func (r *accessLogPolicyReconciler) findTargetRef(ctx context.Context, alp *anv1alpha1.AccessLogPolicy) (client.Object, error) {
 	targetRefNamespacedName := types.NamespacedName{
 		Name:      string(alp.Spec.TargetRef.Name),
 		Namespace: k8s.NamespaceOrDefault(alp.Spec.TargetRef.Namespace),
```

Comment:
namespace should be same as alp itself based on GEP, also empty namespace should not fallback to default